### ISSUE TITLE
`chat` method crash with `searchQueriesOnly=true` parameter

### ISSUE THREAD

**Issue by exentrich** [cohere-ai/cohere-typescript#103]
```js
await cohere.chat({
    message: 'how to create a masonry layout in css?',
    searchQueriesOnly: true
})
````

Crash with error:
```
error: response: Missing required key "text"; response: Missing required key "generation_id"
    at new ParseError (node_modules/cohere-ai/core/schemas/builders/schema-utils/ParseError.js:7:8)
```

It seems like an internal library error. The `chat()` method expects a `text` and `generation_id` in the response, but these keys are missing when using the `searchQueriesOnly` parameter.

**Comment by billytrend-cohere** [comment]
Hey @exentrich, we currently do client side response validation which I think we will turn off because it causes issues like you are seeing. I'll try release a new sdk version today :)

**Comment by exentrich** [comment]
Thank you so much! Cohere is currently the most user-friendly solution. You don't need a PhD to start creating an AI-powered search :) 
Looking forward for the update!

**Comment by billytrend-cohere** [comment]
This is so great to hear! also your issue should be fixed with 7.4.0. Let us know if you have any further problems
